intro-HPC/ch_MATLAB.tex

```diff
@@ -105,6 +105,14 @@ \subsection{Memory issues during compilation}
 %\shellcmd{export \_JAVA\_OPTIONS="-Xmx512M"}%
 \end{prompt}
 
+\subsection{Figures}
+
+figure()
+
+wordt
```

Comment:
english?

Reply:
Ah, crap, that shouldn't have been in here... Will remove & force push.